src/controls/splash.js

```diff
@@ -5,7 +5,6 @@ const Splash = function Splash(options = {}) {
   const defaultContent = '<p></p>';
```

Comment:
There is actually no need to add:
``` 
<p></p> 
```
as default. I would suggest an empty string.

Reply:
Sounds good.